addons/account/static/tests/tours/reconciliation.js

```diff
@@ -25,6 +25,10 @@ Tour.register('bank_statement_reconciliation', {
             extra_trigger: '.o_reconciliation_line:first[data-mode="match_rp"]',
             trigger: '.o_reconciliation_line:nth-child(4) .cell_label:contains("First")'
         },
+        {
+            content: "select the right line to match",
+            trigger: '.o_reconciliation_line:nth-child(4) .o_notebook .cell_label:contains("2019/0001")'
```

Comment:
Strange to see an "add" diff in the tour without something removed... are you sure about this part?

Reply:
Yes, the line is not matched, only proposed because the label is something like `Payment for 2019/0001` and not `Payment for INV/2019/0001`